### DISCUSSION TITLE
软件页面打不开

### DISCUSSION THREAD

**Discussion by sz10280022** [community/community#50984] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

这个软件太差了，给CHAT-GPT提鞋当孙子都不配，今天连软件都打不开，软件加载失败，该网站也不稳定时常打不开，如有回复请至sz10280022@163.com

**Comment by queenofcorgis** [comment]
Hi @sz10280022 , thanks for participating and sharing your feedback!

We take our [Code of Conduct](https://github.com/community/community/blob/0ca37cf594e438fd4aec3c7f3736b95e6b0b3567/CODE_OF_CONDUCT.md) very seriously and want to help ensure that everyone has a good experience free of antagonism and harassment. Unfortunately, we don’t currently have moderators for languages other than English. Until that changes, we need to ask that everyone use English here in the GitHub Community when posting. We’ll be locking any posts in languages other than English for now, including this one.
